modules/osd-create-cluster-ccs.adoc

```diff
@@ -135,19 +135,35 @@ endif::osd-on-gcp[]
 .. Select a cloud provider region from the *Region* drop-down menu.
 .. Select a *Single zone* or *Multi-zone* configuration.
 .. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
+ifdef::osd-on-gcp[]
+. Optional: Expand *Advanced Encryption* to make changes to encryption settings.
+
+.. Select *Use Custom KMS keys* to use custom KMS keys. If you prefer not to use custom KMS keys, leave the default setting *Use default KMS Keys*.
++
+With *Use Custom KMS keys* selected:
```

Comment:
With **Use Custom KMS keys** selected, the IAM service account `osd-ccs-admin` needs to be granted the role **Cloud KMS CryptoKey Encrypter/Decrypter** to the KMS key (e.g. see [Google Cloud Key Management Service guides](https://cloud.google.com/kms/docs/iam#kms-add-iam-member-gcloud)), so suggest to add a note telling it, thanks!

Reply:
Thanks @jianli-wei, info has been added. Thanks for the QE review!